k8s/setup_k8s_cluster.sh

```diff
@@ -60,7 +60,9 @@ username=$1
 : ${k8sVer:=v1.1.4}
 
 # contiv version
-: ${contivVer:=v0.1-05-19-2016.08-34-56.UTC}
+: ${CONTIVNETVERSION?"Please set the Contiv Network version"}
```

Comment:
There is no need for a new variable here. contivVer can be overridden either on command line or in shell. We also don't want to force people to set it if they are ok with the default.